UserInterface.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+    /// <summary>
+    /// User interraction of the application.
+    /// </summary>
+    static public class UserInterface
```

Comment:
Why is this class static? There is no need for that.

Reply:
Removed static from class.